firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+KeyCommands.swift

```diff
@@ -56,15 +56,10 @@ extension BrowserViewController {
                                      method: .press,
                                      object: .keyCommand,
                                      extras: ["action": "reload"])
-        guard let tab = tabManager.selectedTab else { return }
-        if isToolbarRefactorEnabled {
-            store.dispatchLegacy(GeneralBrowserAction(windowUUID: windowUUID,
-                                                      actionType: GeneralBrowserActionType.reloadWebsite))
-        } else {
-            if !contentContainer.hasAnyHomepage {
-                tab.reload()
-            }
-        }
+        guard tabManager.selectedTab != nil else { return }
```

Comment:
Is there any case the selected tab would be nil? And there was a check for the legacy toolbar that we were not on the homepage, do we need this check for the new toolbar too? Same question for the other keyboard shortcuts below

Reply:
You're right. We are checking for selected tab when before reloading anyway so removing the guards. 
The tab reload method checks that we are not reloading a homepage so we don't need to check for it here.